src/coreclr/versionfallback/_version.h

```diff
@@ -0,0 +1,29 @@
+#ifndef VER_COMPANYNAME_STR
```

Comment:
It would be nice if we have just one `_version.h` and one `version.c` in arcade, next to https://github.com/dotnet/arcade/blob/main/src/Microsoft.DotNet.Arcade.Sdk/tools/NativeVersion.rc, and delete all nativeversion.rc, version.h and version.c from the repo (mono, e.g. uses nativeversion.rc out of arcade on windows, but all other subsets have their own copies). AFAICT, we have not really used, changed or taken advantage of any customization since the inception on .NET Core. Also, there are just too many copies of `*[Vv]ersion*` files that get copied under `artifacts/obj` after a full `buid.sh` on linux/mac or `build.cmd` on win with pretty much the same content, which ideally should have just one copy per build at root of obj: `artifacts/obj/version.{h,c,rc}` (that does not differ in values across subsets _during_ any given build).